frontend/src/js/components/devices/widgets/FilterItem.tsx

```diff
@@ -52,15 +52,13 @@ export const FilterItem = ({ attributes, onChange, onSelect, plan, reset }) => {
     onChange({ key, operator, scope, value });
     timer.current = setTimeout(
       () =>
-        key && (value || operator.includes('exists'))
-          ? onSelect({
-              key,
-              operator,
-              scope,
-              value
-            })
-          : null,
-      TIMEOUTS.threeSeconds
+        onSelect({
+          key,
+          operator,
+          scope,
+          value
+        }),
+      TIMEOUTS.debounceDefault
```

Comment:
I guess there is a small chance we get pushback on this for users losing their value input if they intend to change the attribute - but considering the direct attribute change works: 👍